libraries/plugins/apis/database_api/database_api.cpp

```diff
@@ -1422,6 +1423,47 @@ DEFINE_API_IMPL( database_api_impl, verify_signatures )
    return result;
 }
 
+DEFINE_API_IMPL( database_api_impl, find_transaction )
```

Comment:
I am assuming this implementation is WIP because it does not conform to the documented behavior at all.

Reply:
What is WIP?

Reply:
WIP = Work In Progress